src/applications/mhv-supply-reordering/config/form.js

```diff
@@ -63,29 +71,62 @@ const chapters = {
     pages: {
       contactInformation: {
         path: 'contact-information',
-        title: 'Contact information',
+        title: 'Contact information ',
         uiSchema: contactInformation.uiSchema,
         schema: contactInformation.schema,
+        onNavForward: ({ goPath }) => {
+          goPath(PAGE_PATH.REVIEW_AND_SUBMIT);
+        },
       },
       editEmailAddress: {
-        title: 'Edit email address',
-        taskListHide: true,
+        title: 'Email address',
         path: 'edit-email-address',
         CustomPage: EditEmail,
-        CustomPageReview: EditEmail,
-        depends: () => false,
-        uiSchema: {},
-        schema: blankSchema,
+        CustomPageReview: null,
+        onNavForward: ({ goPath }) => {
+          goPath(PAGE_PATH.CONTACT_INFORMATION);
+        },
+        onNavBack: ({ goPath }) => {
+          goPath(PAGE_PATH.CONTACT_INFORMATION);
+        },
```

Comment:
It's so interesting in terms of what goes where when working w/ the form system.